gcp_csi_secrets_store.tf

```diff
@@ -0,0 +1,19 @@
+resource "helm_release" "csi_secrets_store" {
+  count = var.csi_secrets_store_enabled ? 1 : 0
+
+  name       = "csi-secrets-store"
+  repository = "https://kubernetes-sigs.github.io/secrets-store-csi-driver/charts"
+  chart      = "secrets-store-csi-driver"
+
+  namespace        = "kube-system"
+  create_namespace = false
+}
+
+data "kubectl_file_documents" "gcp_csi" {
+  content = file("gcp-provider-plugin.yaml")
```

Comment:
Instead of storing the file we should use:

https://registry.terraform.io/providers/hashicorp/http/latest/docs/data-sources/http